### PR TITLE
Improve add target placeholder page

### PR DESCRIPTION
**Pull request by SashaKryzh**
- Removed unused code
- Added ability to add users to the waitlist
- Updated `pnpm` version in the Github Actions

### CODE REVIEW

File: src/ui/GradientContainer.tsx [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
+};
+
+export function GradientContainer(props: GradientProps) {
   return (
-    <div className='bg-gradient-to-br from-[#F1ACAC] via-[#BF98C6] to-[#2833D0] p-[0.20rem]'>
-      <div className='h-full w-full bg-white'>{props.children}</div>
+    <div
+      className={cn(
+        'bg-gradient-to-br from-[#F1ACAC] via-[#BF98C6] to-[#2833D0] p-[0.20rem]',
+      )}
```
Comment (denitdao) [inline src/ui/GradientContainer.tsx]: could be simple class name, but not critical

File: src/server/sendMail.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import nodemailer from 'nodemailer';
+import type Mail from 'nodemailer/lib/mailer';
+
+const transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: process.env.NODEMAILER_EMAIL,
+    pass: process.env.NODEMAILER_PSW,
```
Comment (denitdao) [inline src/server/sendMail.ts:9]: Would like to know how to set these locally. Should we add them to our example.env file or anything?

File: src/server/sendMail.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  const mail: Mail.Options = {
+    ...message,
+    from: process.env.NODEMAILER_EMAIL,
+    bcc: bcc,
+  };
+
+  transporter.sendMail(mail, (err, info) => {
+    console.log(err);
+    console.log(info);
+  });
```
Comment (denitdao) [inline src/server/sendMail.ts]: I suppose user will see 'Дякуємо! Ми повідомимо Вас, коли цей функціонал буде готовий.' as we're just ignoring the error here?

File: src/server/sendMail.ts (hunk 2 above)
Comment (SashaKryzh) [reply src/server/sendMail.ts:9]: Sent you in DM

File: src/ui/GradientContainer.tsx (hunk 1 above)
Comment (SashaKryzh) [reply src/ui/GradientContainer.tsx]: Fixed

File: src/server/sendMail.ts (hunk 3 above)
Comment (SashaKryzh) [reply src/server/sendMail.ts]: Fixed

**Review by denitdao — APPROVED** [review]

**Pull request merged**